cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java

```diff
@@ -127,8 +130,11 @@ public void execute() {
                             packageMdFiles.add(packageMdFile);
                         }
                     } else if (srcInput.equalsIgnoreCase("main") || srcInput.equalsIgnoreCase("m")) {
-                        out.print("Package for the main : (no package) ");
-                        String packageName = scanner.nextLine().trim();
+                        String packageName;
+                        do {
+                            out.print("Package for the main : (no package) ");
```

Comment:
Shall we get rid of the space before the colon ?  Doesnt seem to be consistent with other messages.

Reply:
Fixed with 45ff1fd